packages/react/src/components/FileUploader/FileUploaderDropContainer.js

```diff
@@ -0,0 +1,152 @@
+import React, { useCallback, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { settings } from 'carbon-components';
+import { keys, matches } from '../../internal/keyboard';
+import uniqueId from '../../tools/uniqueId';
+
+const { prefix } = settings;
+
+export default function FileUploaderDropContainer(props) {
+  const inputRef = useRef();
+  const {
+    accept,
+    className,
+    id,
+    disabled,
+    labelText,
+    multiple,
+    name,
+    onAddFiles,
+    role,
+    tabIndex,
+    ...other
+  } = props;
+  const uid = useRef(uniqueId()).current;
```

Comment:
```suggestion
  const uid = useRef(uniqueId());
```

As a convention we should try and treat `.current` as a way to get the latest value at the call site versus using it at the top.